schemas/data/measured-window-inactive.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/metrics/advertising/viewability/measured-window-inactive",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "measured-window-inactive",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/metric"
+  ],
+  "description": "The advertising/viewability metric measured-window-inactive describes the whether at impression time the ad was displayed in a window not active by the user.",
```

Comment:
Typo, remove "the" before "whether"

Can this not be a property of the quantiles metric?